src/flows/components/panel/FlowPanel.tsx

```diff
@@ -36,14 +37,16 @@ import {RefContext} from 'src/flows/context/refs'
 
 export interface Props extends PipeContextProps {
   id: string
+  shouldShow?: boolean
 }
 
 export interface HeaderProps {
   id: string
   controls?: ReactNode
+  shouldShow?: boolean
```

Comment:
this should be a ReactNode, not a boolean, so that we can pass arbitrary buttons that are shown all the time, and not "show the control buttons all the time"

Reply:
Gotcha.